### PR TITLE
#346 Adding info for parsing JSON for srsARN value

### LINKED ISSUE #346: Optimize Bucket Notification ARN collection
Per customer feedback, retrieving the [ARN](https://docs.min.io/minio/baremetal/monitoring/bucket-notifications/publish-events-to-kafka.html#restart-the-minio-deployment) can be difficult if the user does not have access to the subsequent console logs (permissions, access, etc.)

A more reliable method would be to use `mc admin info --json ALIAS` to retrieve the server configuration and parse the JSON for the required ARN. 

via @kannappanr the command `mc admin info ALIAS --json | jq .info.targetARN` should work, though it requires installing `jq`. `info` subs out for the acronym/short-name for each bucket notification target

# Goals

Update Bucket Notification target docs with alternative ARN retrieval instructions

# Non-Goals

Any other changes to bucket notification documentation.

### PR DESCRIPTION
**Pull request by djwfyi**
Closes #346

Adds a new include file for bucket notification ARN identification.
Modifies each `publish` file under `monitoring/bucket-notifications` to add include file and a substitution directive.

### CODE REVIEW

**Review by ravindk89 — CHANGES_REQUESTED** [review]
Minor nits and one open point of discussion.

Also you man need to add `.DS_Store` to the `.gitignore` file, I think that is coming from osx.

File: source/includes/bucket-notifications-find-arn.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.. admonition:: Identifying the ARN for your bucket notifications
```
[the remark this answers is not in the document]
Comment (djwfyi) [reply source/includes/bucket-notifications-find-arn.rst:1]: I can see the logic of doing things in a common file with the `start-after` and `end-before` fences.

File: source/includes/bucket-notifications-find-arn.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+   :class: note
+
+   You defined the target ARN for your bucket notifications when creating the endpoint in step 1.
```
[the remark this answers is not in the document]
Comment (djwfyi) [reply source/includes/bucket-notifications-find-arn.rst:4]: Updating line 4 to

> You defined the name to assign to the target ARN for your bucket notifications when creating the endpoint in step 1.
> The command options below allow you to find the entire target ARN.

File: source/monitoring/bucket-notifications/publish-events-to-elasticsearch.rst [hunk 3]
```diff
@@ -6,6 +6,8 @@ Publish Events to Elasticsearch
 
 .. default-domain:: minio
 
+.. |ARN| replace:: ``arn:minio:sqs::primary:Elasticsearch``
```
[the remark this answers is not in the document]
Comment (djwfyi) [reply source/monitoring/bucket-notifications/publish-events-to-elasticsearch.rst:9]: The capital E in `Elasticsearch` for the resource in the ARN is how the ARN displays throughout the rest of the file.
I did not update all the instances of the ARN in the file to use the substitution.

File: source/includes/common-bucket-notifications.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
+.. start-bucket-notification-find-arn
+
+.. admonition:: Identifying the ARN for your bucket notifications
+   :class: note
+
+   You defined the name to assign to the target ARN for your bucket notifications when creating the endpoint in step 1.
+   The command options below allow you to find the entire target ARN.
```
Comment (ravindk89) [inline source/includes/common-bucket-notifications.rst:7]: "The steps below return all configured ARNs on the deployment. Identify the ARN created in step 1 using the name you specified (``IDENTIFIER``)"

Or something.

File: source/includes/common-bucket-notifications.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+   #. Copy and run the following command, replacing ``ALIAS`` with the :ref:`alias <alias>` of the deployment.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin info --json ALIAS
+
+   #. In the JSON output, look for the key ``info.sqsARN``.
+   
+      The ARN you need is the value of that key, such as |ARN|
```
Comment (ravindk89) [inline source/includes/common-bucket-notifications.rst:20]: Same thought specifying it's the ARN that matches both the name + service as specified in step 1

**Review by ravindk89 — APPROVED** [review]
Two optional nits, otherwise LGTM.

**Review by harshavardhana — CHANGES_REQUESTED** [review]
Please remove the .DS_store file from the commit

**Comment by ravindk89** [comment]
Ah we added that to `.gitignore` in another commit.

Should be fine to add it here, it should all get settled with the rebase

**Review by ravindk89 — APPROVED** [review]
LGTM. If there's some nits or grits left we can deal with them in another pass.

**Pull request merged**